exchange/bidder_test.go

```diff
@@ -230,78 +237,389 @@ func TestConnectionClose(t *testing.T) {
 	}
 }
 
-// TestMultiCurrencies makes sure that bidderAdapter.requestBid returns errors if the bidder pass several currencies in case of multi HTTP calls.
+// TestTryGetConversionRate
+func TestTryGetConversionRate(t *testing.T) {
+	// TODO:
```

Comment:
Should either do this or delete the empty test.

Reply:
Yes, sorry about that, just removed. It was meant to test a bidder function to get the conversion rate, but not needed anymore since the API is clear enough, no need to encapsulate more.